### ISSUE TITLE
cgroup EROFS not handled correctly

### ISSUE THREAD

**Issue by richiejp** [linux-test-project/ltp#1082]
clone303 fails with TBROK when `/sys/fs/cgroup` is mounted as read-only (ro) which is normal for a container. We probably want to handle this with TCONF.

```
# clone303
tst_buffers.c:56: TINFO: Test is using guarded buffers
tst_cgroup.c:387: TBROK: mkdirat(4</sys/fs/cgroup>, 'ltp', 0777): EROFS (30)

Summary:
passed   0
failed   0
broken   1
skipped  0
warnings 0
```

Can be reproduced with the Alpine container:
https://github.com/richiejp/ltp/blob/alpine-container/Containerfile
https://hub.docker.com/repository/docker/richiejp/ltp/general

**Comment by Martchus** [comment]
https://patchwork.ozlabs.org/project/ltp/list/?series=374745

**Comment by Martchus** [comment]
The patch has been merged so this issue can be closed.

**Comment by pevik** [comment]
We both forget to add `Fixes: #1082` to the git commit message, thus closing manually.
Fixed in ec08f1d382ef87e7c353e4720d17050d068f577f.
